pkg/daemon/update.go

```diff
@@ -83,6 +83,16 @@ func getNodeRef(node *corev1.Node) *corev1.ObjectReference {
 	}
 }
 
+type drainLogger struct{}
+
+func (dl *drainLogger) Logf(format string, v ...interface{}) {
+	glog.Infof(format, v)
```

Comment:
`make verify` is failing, I think this should be something like:

glog.Infof(format, v...)

Reply:
aaahhhhh

Reply:
updated